week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,55 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10; // returns a random integer from 0 to 9
```

Comment:
You're almost there, so Math.random return a float number (float mean with decimal) between 0 and 0.99999...
meaning the function will return from 0 to 9.99999...